### PR TITLE
Adds S.E.L.F faction to infiltrators

### PR DESCRIPTION
**Pull request by ReinaCoder**
They have an objective to steal a functional AI, subvert a borg, and the bog standard kill obj.
They start with a specialized one time use emag that only works on borgs for their objective, along with an intellicard for stealing an AI.
Adds some more variety in infiltrators.
fxKqZZEpHy
(special emag pictured)
thx 2  all the fulp contribs

Also this PR removes some unnecessary infil code like giving gorlex and emag, the code for their hijack along with changing their suit from orange to red.

### CODE REVIEW

File: fulp_modules/features/antagonists/infiltrators/infil_items.dm [hunk 1]
```diff
@@ -143,4 +143,35 @@
...
+	name = "single-use silicon cryptographic sequencer"
+	desc = "A specialized cryptographic sequencer used to free cyborgs. Will become voided after a one time use."
+	icon = 'fulp_modules/features/antagonists/infiltrators/icons/infils.dmi'
+	icon_state = "cyborg_hack"
+    //whether we have successfully emagged a borg
+	var/used = FALSE
+
+/obj/item/card/emag/silicon_hack/proc/use_charge(mob/user)
+    used = TRUE
+    to_chat(user, span_boldwarning("You use [src], and it interfaces with the cyborg's panel."))
```
Comment (Pepsilawn) [inline fulp_modules/features/antagonists/infiltrators/infil_items.dm:156]: U should use balloon alerts upvote if u agree

Comment (Pepsilawn) [reply fulp_modules/features/antagonists/infiltrators/infil_items.dm:156]: Why are you setting used to TRUE here

Comment (Pepsilawn) [reply fulp_modules/features/antagonists/infiltrators/infil_items.dm:156]: Nvm

**Comment by SgtHunk** [comment]
pretty cool

**Pull request merged**